Source/ASDisplayNode+Beta.h

```diff
@@ -113,6 +113,19 @@ typedef struct {
 #endif
 
 /**
+ * @abstract Whether this node acts as an accessibility container. If set to YES, then this node's accessibility label will represent
+ * an aggregation of all child nodes' accessibility labels. Nodes in this node's subtree that are also accessibility containers will
+ * not be included in this aggregation, and will be exposed as separate accessibility elements to UIKit.
+ */
+@property (nonatomic, assign) BOOL isAccessibilityContainer;
```

Comment:
1. Just acknowledging that while this is not Objective-C convention (having `is` in the property name), this follows the UIAccessibility convention so good call.

1. Should we use objective-C atomics here? Not a big deal either way.